matter/wifi/rs911x/rsi_if.c

```diff
@@ -158,83 +161,85 @@ static void wfx_rsi_ipchange_cb(uint16_t status, uint8_t *buf, uint32_t len)
 /*
  * Got RAW WLAN data pkt
  */
-static void wfx_rsi_wlan_pkt_cb(uint16_t status, uint8_t *buf, uint32_t len)
-{
+static void wfx_rsi_wlan_pkt_cb(uint16_t status, uint8_t *buf, uint32_t len) {
   // WFX_RSI_LOG("%s: status=%d, len=%d", __func__, status, len);
   if (status != RSI_SUCCESS) {
     return;
   }
   wfx_host_received_sta_frame_cb(buf, len);
 }
 #endif /* !Socket support */
-static int32_t wfx_rsi_init(void)
-{
+static int32_t wfx_rsi_init(void) {
   int32_t status;
-  uint8_t buf[128];
+  uint8_t buf[BUF_SIZE_128];
   extern void rsi_hal_board_init(void);
 
-  /* 
+  /*
    * Get the GPIOs/PINs set-up
    */
-  //rsi_hal_board_init ();
+  // rsi_hal_board_init ();
   WFX_RSI_LOG("%s: starting(HEAP_SZ = %d)", __func__, SL_HEAP_SIZE);
   //! Driver initialization
   status = rsi_driver_init(wfx_rsi_drv_buf, WFX_RSI_BUF_SZ);
-  if ((status < 0) || (status > WFX_RSI_BUF_SZ)) {
-    WFX_RSI_LOG("%s: error: RSI drv init failed with status: %02x", __func__, status);
+  if ((status < RSI_DRIVER_STATUS) || (status > WFX_RSI_BUF_SZ)) {
+    WFX_RSI_LOG("%s: error: RSI drv init failed with status: %02x", __func__,
+                status);
     return status;
   }
 
   WFX_RSI_LOG("%s: rsi_device_init", __func__);
   //! Redpine module intialisation
   if ((status = rsi_device_init(LOAD_NWP_FW)) != RSI_SUCCESS) {
-    WFX_RSI_LOG("%s: error: rsi_device_init failed with status: %02x", __func__, status);
+    WFX_RSI_LOG("%s: error: rsi_device_init failed with status: %02x", __func__,
+                status);
     return status;
   }
   WFX_RSI_LOG("%s: start wireless drv task", __func__);
   /*
    * Create the driver task
    */
-  if (xTaskCreate((TaskFunction_t)rsi_wireless_driver_task, "rsi_drv", WFX_RSI_WLAN_TASK_SZ, NULL, 1, &wfx_rsi.drv_task)
-      != pdPASS) {
+  if (xTaskCreate((TaskFunction_t)rsi_wireless_driver_task, "rsi_drv",
+                  WFX_RSI_WLAN_TASK_SZ, NULL, TASK_PRIORITY_1, &wfx_rsi.drv_task) != pdPASS) {
```

Comment:
`WLAN_TASK_PRIORITY` would be more appropriate I believe

Reply:
changed